docs/content/guides/concurrency-scheduling.md

```diff
@@ -0,0 +1,240 @@
+of which is defined by its priority and weight. This classification is crucial
+for the request scheduling process.
+
+The scheduler priorities request admission based the priority and weight
+assigned to the corresponding workload. This mechanism ensures that
+high-priority requests are handled appropriately even under high load.
+
+:::note Pre-Requisites
+
+Before exploring Aperture's concurrency scheduling capabilities, make sure that
+you have signed up to [Aperture Cloud](https://app.fluxninja.com/sign-up) and
+set up an organization. For more information on how to sign up, follow our
+[step-by-step guide](/reference/cloud-ui/sign-up.md).
+
+:::
+
+## Concurrency Scheduling with Aperture SDK
+
+The first step to using the Aperture SDK is to import and set up Aperture
+Client:
+
+```mdx-code-block
+<Tabs>
+  <TabItem value="TypeScript">
+```
+
+<CodeSnippet lang="ts" snippetName="clientConstructor" />
+
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
+
+You can obtain your organization address and API Key within the Aperture Cloud
+UI by clicking the `Aperture` tab in the sidebar menu.
+
+The next step consists of setting up essential business labels to prioritize
+requests. For example, requests can be prioritized by user tier classifications:
+
+```mdx-code-block
+<Tabs>
+  <TabItem value="Typescript">
+```
+
+<CodeSnippet lang="ts" snippetName="Priority" />
+
+```mdx-code-block
+ </TabItem>
+</Tabs>
+```
+
+The next step is making a `startFlow` call to Aperture. For this call, it is
+important to specify the control point (`concurrency-scheduling-feature` in our
+example) and the labels that will align with the concurrency scheduling policy.
+The `priority` label is necessary for request prioritization, while the
+`workload` label differentiates each request.
+
+According to the policy logic designed to prevent service limit breaches,
+Aperture will, on each `startFlow` call, either give precedence to a critical
+request or queue a less urgent one when approaching API limits. The duration a
+request remains in the queue is determined by the gRPC deadline, set within the
+`startFlow` call. Setting this deadline to `120000` milliseconds, for example,
+indicates that the request can be queued for a maximum of 2 minutes. After this
+interval, the request will either be processed or discarded, depending on its
+position in the queue.
+
+In this example, we're only tracking and logging requests sent to Aperture.
+However, once the `startFlow` call is made, business logic can be executed
+directly, as excess requests are queued, eliminating the need to check if a flow
+`shouldRun`.
+
+```mdx-code-block
+<Tabs>
+  <TabItem value="TypeScript">
+```
+
+<CodeSnippet lang="ts" snippetName="CSStartFlow" />
+
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
+
+It is important to make the `end` call made after processing each request, in
+order to send telemetry data that would provide granular visibility for each
```

Comment:
Apart from telemetry data, the end call is also used for removing in flight requests